pkg/securestorage/securestorage.go

```diff
@@ -144,7 +145,7 @@ func (s *SecureStorage) openKeyring() (keyring.Keyring, error) {
 	}
 
 	// enable debug logging if the verbose flag is set in the CLI
-	if debug.CliVerbosity == debug.VerbosityDebug {
+	if debug.Debug {
```

Comment:
This can be replaced with the following after https://github.com/common-fate/clio/pull/4 is merged
```suggestion
	if clio.IsDebug() {
```